optimizely/client/factory.go

```diff
@@ -85,3 +87,28 @@ func (f OptimizelyFactory) ClientWithContext(ctx context.Context) (*OptimizelyCl
 
 	return nil, fmt.Errorf("Cannot create ClientWithContext")
 }
+
+// Client returns a client initialized with the defaults
+func (f OptimizelyFactory) Client() (*OptimizelyClient, error) {
+	var configManager optimizely.ProjectConfigManager
+
+	if f.SDKKey != "" {
+		url := fmt.Sprintf(CDNTemplate, f.SDKKey)
+		request := config.NewRequester(url)
+		ctx := context.Background()
+		var cancel context.CancelFunc
+		ctx, cancel = context.WithCancel(ctx)
```

Comment:
Should we pass the `cancelFunc` to the client in the other factory method where the user passes in context? They should be able to call `optimizelyClient.Stop()` with the client created `withContext` as well